lib/simctl.js

```diff
@@ -63,6 +63,8 @@ const DEFAULT_OPTS = {
  * command input and outputs.
  * @property {string|string[]} [architectures] - One or more architecture names to be enforced while
  * executing xcrun. See https://github.com/appium/appium/issues/18966 for more details.
+ * @property {number} [timeout] - The maximum number of milliseconds
+ * to wait for single synchronous xcrun command over the execTimeout by SimctlOpts.
```

Comment:
over the execTimeout by SimctlOpts -> If not provided explicitly then the value of `execTimeout` property is used by default